tests/config/config.d/logger_test.xml

```diff
@@ -1,5 +1,5 @@
 <clickhouse>
     <logger>
-        <level>debug</level>
+        <level>test</level>
```

Comment:
This is just to test on the CI and then we'll revert before merging, right?

Reply:
This commit is a revert. The `test` is the original value